webapp/src/Service/ImportExportService.php

```diff
@@ -823,8 +823,9 @@ public function importAccountsJson(array $data, ?string &$message = null, ?array
             } else if (!in_array($type, $djRoles)) {
```

Comment:
I think the problem is that we now want the keys instead of the values.
```suggestion
            } else if (!in_array($type, array_keys($djRoles))) {
```

Reply:
Right, I realized the same. Tests succeed now locally, let's see what CI finds...